CMakeLists.txt

```diff
@@ -51,13 +47,15 @@ target_include_directories(
     PUBLIC
     "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>"
     "$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>"
+    "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>" # ?? although your exampleTargets.cmake has a INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/example"
```

Comment:
This is handled in the install section of the project CML with the [`INCLUDES DESTINATION`](https://cmake.org/cmake/help/latest/command/install.html#targets) argument:

> This option specifies a list of directories which will be added to the INTERFACE_INCLUDE_DIRECTORIES target property of the <targets> when exported by the install(EXPORT) command. If a relative path is specified, it is treated as relative to the $<INSTALL_PREFIX>.

This makes it easy to keep concerns in separate sections of the CML instead of spaghettifying everything together.